lib/index.js

```diff
@@ -85,25 +95,82 @@ Vero.prototype.track = function(payload, fn) {
  * @api public
  */
 
-Vero.prototype.alias = request('/users/reidentify', 'put');
+Vero.prototype.alias = request('alias', '/users/reidentify', 'put');
+
+/**
+ * Add or Remove Tags.
+ *
+ * http://developers.getvero.com/?bash#tags
+ *
+ * @param {Facade} msg
+ * @param {Object} tags
+ * @param {Function} fn
+ * @api public
+ */
+
+Vero.prototype.addOrRemoveTags = function(msg, tags, fn) {
+  // Check to ensure tags object is properly formatted.
+  var err;
+  if (Object.prototype.toString.call(tags) !== '[object Object]') {
+    err = new Error('Tag option must be passed as an Object.');
+  }
+
+  if (!tags.action || tags.action !== 'add' && tags.action !== 'remove') {
+    err = new Error('Tags action value must be either \'add\' or \'remove\'');
+  }
+
+  if (!tags.values || !Array.isArray(tags.values)) {
+    err = new Error('Could not detect tag values. Tag values must be passed as array of strings.');
+  }
+
+  if (err instanceof Error) {
+    err.status = 400;
+    return fn(err);
```

Comment:
Would it make sense to instead check these cases in an `ensure` function, and reject the whole message?